### PR TITLE
VSC and USDV token updated

### PR DESCRIPTION
**Pull request by abinav124**
Hi,
We have committed the VSC and USDV token into Vechain Mainnet.

### CODE REVIEW

File: tokens/main/0x094042f9719cd6736fa3bd45b605b1b2a23abdec/additional.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "website":"https://www.odee.com/ ", 
+  "links":{
+      "twitter":"https://twitter.com/VyvoSmartChain",      
+      "telegram":"https://t.me/odeeofficial",
+      "medium":"https://medium.com/@VyvoSmartChain"
+    },
+  "whitePaper":""
```
Comment (libotony) [inline tokens/main/0x094042f9719cd6736fa3bd45b605b1b2a23abdec/additional.json:8]: This line should be removed.

**Review by libotony — CHANGES_REQUESTED** [review]

File: tokens/main/0x094042f9719cd6736fa3bd45b605b1b2a23abdec/additional.json (hunk 1 above)
Comment (abinav124) [reply tokens/main/0x094042f9719cd6736fa3bd45b605b1b2a23abdec/additional.json]: Those lines its removed

Comment (abinav124) [reply tokens/main/0x094042f9719cd6736fa3bd45b605b1b2a23abdec/additional.json]: committed ID [5265deb](https://github.com/vechain/token-registry/pull/122/commits/5265deb77577827f41528ee397709000390b2790)

File: tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/info [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (libotony) [inline tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/info:1]: This file should be named `info.json`

**Review by libotony — CHANGES_REQUESTED** [review]

File: tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/info (hunk 2 above)
Comment (abinav124) [reply tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/info]: The File name changed

Comment (abinav124) [reply tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/info]: Committed ID [cd568c4](https://github.com/vechain/token-registry/pull/122/commits/cd568c4412419018746829aa77298bb34b70df6d)

File: tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/additional.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "website": "https://www.vyvo.org/",
+  "links": {
+    "twitter": "https://twitter.com/VyvoSmartChain",
+    "medium": "https://medium.com/@VyvoSmartChain",
```
Comment (libotony) [inline tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/additional.json:5]:
```suggestion
    "medium": "https://medium.com/@VyvoSmartChain"
```

**Review by libotony — CHANGES_REQUESTED** [review]

File: tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/additional.json (hunk 3 above)
Comment (abinav124) [reply tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/additional.json]: Its updated

Comment (abinav124) [reply tokens/main/0x65c542ad413dd406d7ae5e47f61fbda027ce7983/additional.json]: Committed ID [6097df2](https://github.com/vechain/token-registry/pull/122/commits/6097df2f01e0eb723d523b36373b40c4fc46f305)

**Review by libotony — APPROVED** [review]

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**